features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageWireframeHelper.kt

```diff
@@ -55,6 +57,25 @@ internal class ImageWireframeHelper(
         val displayMetrics = view.resources.displayMetrics
         val applicationContext = view.context.applicationContext
         val mimeType = imageCompression.getMimeType()
+        val density = displayMetrics.density
+
+        // in case we suspect the image is PII, return a placeholder
+        @Suppress("UnsafeCallOnNullableType") // drawable already checked for null in isValid
+        if (imageTypeResolver.isDrawablePII(
+                drawableProperties.drawable!!,
```

Comment:
minor thought: maybe we can avoid this `!!` usage? I can see that `DrawableProperties#isValid` above is checking that drawable is not null, but compiler doesn't know about it and moreover `drawable` getter may do something different.

Maybe we can get rid of `isValid` property and checking things here? This way compiler will know that `drawable` is not null. Or even change the type of `drawable` in `DrawableProperties` to be non-null (is `DrawablePropeties` useful without drawable)?

this is the only usage of `!!` in production codebase and this is a bit confusing.

Reply:
I'll refactor the code to make DrawableProperties drawable non-nullable